cvat/apps/dataset_manager/task.py

```diff
@@ -453,8 +453,25 @@ def _init_shapes_from_db(self):
         for shape_id, shape_elements in elements.items():
             shapes[shape_id].elements = shape_elements
 
-        serializer = serializers.LabeledShapeSerializer(list(shapes.values()), many=True)
-        self.ir_data.shapes = serializer.data
```

Comment:
Unnessesary converstion to serializer and back, from performance point of view it would be better to handle raw data, received from the database, keeping output